### ISSUE TITLE
Feature Request: support DNS3 and DNS4 items for IPv6 upstream DNS servers

### ISSUE THREAD

**Issue by nickbreen** [pi-hole/docker-pi-hole#262]
setupVars.conf supports up to four custom upstream DNS servers, two for IPv4 and two for IPv6.

E.g. setupVars.conf with IPv4 and IPv6 Google DNS addresses
```
PIHOLE_DNS_1=8.8.8.8
PIHOLE_DNS_2=8.8.4.4
PIHOLE_DNS_3=2001:4860:4860::8888
PIHOLE_DNS_4=2001:4860:4860::8844
```

Add support for _3 and _4 in `bash_functions.sh` and `start.sh`.

**Comment by volschin** [comment]
Would be really important to allow this, because information is lost on every container recreation.

**Comment by hniemann** [comment]
+1

**Comment by sertione** [comment]
need this feature too

**Comment by IngmarStein** [comment]
If you only need to specify one DNS server with its IPv4 and IPVv6 addresses, you can just use `DNS1` and `DNS2`. IPv6 servers don't need to go into `DNS3` or `DNS4`. The UI will display an IPv6 server you specify as `DNS2` in the `Custom 3 (IPv6)` field.

**Comment by PromoFaux** [comment]
Closed in place of the pinned #720
